scripts/contacts/migrateContactsV1toV2.js

```diff
@@ -33,6 +34,18 @@ const getCreatedByApp = contact => {
   return null
 }
 
+const filterContact = contact => {
+  /* eslint-disable no-unused-vars */
+  let { vendorId, metadata, ...filteredContact } = contact
+  let { version, cozy, ...filteredMetadata } = metadata
```

Comment:
FYI

```
const omit = require('lodash/omit')
const filteredContact = omit(contact, 'vendorId', 'metadata')
```

does the same. IMHO it's clearer

Reply:
yeah, I tried both of them. The thing is that after the omit, I still have to get `metadata` to remove it if it's empty. Maybe it's still better than this though